typescript/looker/rtl/apiMethods.ts

```diff
@@ -41,6 +41,18 @@ export class APIMethods {
     this.apiPath = authSession.settings.base_url === '' ? '' : `${authSession.settings.base_url}/api/${this.apiVersion}`
   }
 
+  /** A helper method for constructing with a type as a param
+   *
+   * Example:
+   *   APIMethods.create(Looker40SDK, authSession)
+   *
+   * @param type
+   * @param authSession
+   */
+  static create<T extends APIMethods>(type: (new (authSession: IAuthSession) => T), authSession: IAuthSession): T {
+    return new type(authSession)
```

Comment:
This fn isn't used by this PR, and the example rings a little hollow, since the real use case is when the caller doesn't have a concrete sdk type, just passing through a type param.  Can be deleted if desired.